components/Dialogs/src/MDCAlertController.m

```diff
@@ -367,6 +374,10 @@ - (void)setupAlertView {
   self.alertView.messageFont = self.messageFont;
   self.alertView.titleColor = self.titleColor;
   self.alertView.messageColor = self.messageColor;
+  if (self.backgroundColor) {
+    // Avoid reset background color to transparent when self.backgroundColor is nil.
```

Comment:
Lets make the backgroundColor property get the self.alertView.background color and not even hold a value in this class.

Or initialize backgroundColor property to the same value that the alertView's background color.

Reply:
self.alertView may not exist when the background color is set.  This is following the same approach as other properties that affect the view - the value is saved as an alert property, and assigned to the alertView in this  method (setupAlertView) as the alertView materializes.